api/routes/user.js

```diff
@@ -236,7 +236,7 @@ router.post('/users', function (req, res) {
   } else if (!checkIfTokenValid(req)) {
     return res.sendStatus(UNAUTHORIZED)
   }
-  User.find({}, function (error, result) {
+  User.find({}, null, { sort: { joinDate: -1 } }, function (error, result) {
```

Comment:
my man if u wanna sort do it [like this](https://github.com/SCE-Development/Core-v4/blob/1667f5a2a93afceedafa6c70f66f3581cca9aa26/api/routes/Event.js#L20)